### PR TITLE
pagure: errors may be not set, add tests

### LINKED ISSUE #334: improve pagure unit tests
edited by @lachmanfrantisek :

TODO:
- Add tests for https://github.com/packit-service/ogr/pull/333 (better pagure error messages)
  - e.g. wrong URL, wrong token, not-found object,...

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #334

### CODE REVIEW

**Comment by TomasTomecek** [comment]
VCRpy and requre clearly not play well together

File: tests/integration/test_token_is_none_then_set [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      message: UNAUTHORIZED
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - token asdqwe
```
Comment (lachmanfrantisek) [inline tests/integration/test_token_is_none_then_set:61]: You manually changed that?

**Comment by lachmanfrantisek** [comment]
> VCRpy and requre clearly not play well together

Looks like that -- you need to create a new folder to not interfere the imports.

File: tests/integration/test_token_is_none_then_set (hunk 1 above)
Comment (TomasTomecek) [reply tests/integration/test_token_is_none_then_set:61]: yes

**Comment by TomasTomecek** [comment]
Closing this in the meantime and will fix the issue in a separate PR.

**Pull request closed without merging**